docs/src/how-to/5-contribute-to-koma.md

```diff
@@ -36,39 +36,56 @@ Now, you need to ensure that your GitHub account is connected to VSCode. This al
 - Sign in to your GitHub account if you're not already signed in.
 
 >💡You can also check if your `git` credentials are correctlly added to your machine by writing in the VScode terminal:
-```shell
-git config --global user.name
-git config --global user.email
-```
+>```shell
+>git config --global user.name
+>git config --global user.email
+>```
 
 ### 4. Open your forked repository in VSCode
 
 In VSCode, click on **File** -> **Open Folder...** and select your `~/.julia/dev/KomaMRI/` directory (`C:\Users\<user-name>\.julia\dev\KomaMRI\` if you are using Windows).
 
-![](../assets/open-folder.png)
-
 Now add the fork URL by clicking **Source Control** -> **...** -> **Remote** -> **Add Remote...**
 
-![](../assets/add-remote.png)
+<img width="100%" src="../assets/add-remote.png"/>
 
 This will create the option to provide a repository URL. Here is where you will paste your fork URL and give it the name `my-fork`.
 
-![](../assets/create-remote-step1.png)
-![](../assets/create-remote-step2.png)
+<img width="100%" src="../assets/create-remote-step1.png"/>
+<img width="100%" src="../assets/create-remote-step2.png"/>
 
 >💡Press `Yes` when prompted to constantly fetch in the future.
 
 The Julia extension should automatically detect the `KomaMRI` environment. To check this, look at the status bar (bottom) end you should see `Julia env: KomaMRI`. If this is not the case, click the option in the menu bar and select KomaMRI.jl.
 
 ### 5. KomaMRI monorepo setup
 
-As KomaMRI.jl contains multiple packages in one GitHub repository, you need to specify that you want to use your local copies (instead of the ones available on the Julia registries) with:
+As KomaMRI.jl contains multiple packages in one GitHub repository, you need to specify that you want to use your local copies (instead of the ones available on the Julia registries) and using the `instantiate` command to install all the required packages (specified in `project.toml`) with the following script:
+
 ```shell
-(KomaMRI) pkg> dev ./KomaMRIBase ./KomaMRICore ./KomaMRIFiles ./KomaMRIPlots
+pkg> activate KomaMRICore
+pkg> dev ./KomaMRIBase
+pkg> instantiate
+
+pkg> activate KomaMRIFiles
+pkg> dev ./KomaMRIBase
+pkg> instantiate
+
+pkg> activate KomaMRIPlots
+pkg> dev ./KomaMRIBase
+pkg> instantiate
+
+pkg> activate .
+pkg> dev ./KomaMRICore ./KomaMRIFiles ./KomaMRIPlots
+pkg> instantiate
 ```
-Finally, use the `instantiate` command to install all the required packages (specified in the `Project.toml`):
+In case you want to contribute specifically in documentation, you will need to use the `docs` enviroment with the following script:
+
 ```shell
-(KomaMRI) pkg> instantiate
+pkg> activate docs
+pkg> dev ./KomaMRI
+pkg> instantiate
+
 ```
```

Comment:
Was the instantiate necessary? There is an additional new line after the las instantiate. The command to setup the monorepo is too long, put inside a `!!! details` callout, so it is collapsable.

Reply:
It wasn't necessary. Corrected in commit [a26a42c](https://github.com/JuliaHealth/KomaMRI.jl/pull/510/commits/a26a42c9cb791e0cddcb51b59c2dbe283a0dcf81)

Reply:
This is not solved yet.

Reply:
Fixed in commit [3c53c21](https://github.com/JuliaHealth/KomaMRI.jl/pull/510/commits/3c53c21b3b9a3b788e92f7508e263efc56a9c83d)